### PR TITLE
[AAP-7709] Add notifications list

### PR DESCRIPTION
**Pull request by skochay330**
This PR adds the Notifiers page (add/edit pages have not yet been built) 

Jira issue: https://issues.redhat.com/browse/AAP-7709 

Before:
Screenshot 2024-03-06 at 4 54 18 PM

After: 
Screenshot 2024-03-21 at 5 33 49 PM

### CODE REVIEW

**Comment by AlexSCorey** [comment]
Lets make the side bar label, and the List page title the same.
Screenshot 2024-03-07 at 9 09 19 AM

**Comment by AlexSCorey** [comment]
The edit and copy buttons show be pinned to always show on the list item.  This is convention on other lists in controller.

Screenshot 2024-03-07 at 9 09 31 AM

**Comment by AlexSCorey** [comment]
The status on the list item should be a label rather than just an icon and text.
https://tower-mockups.testing.ansible.com/patternfly/notifications/notifications/

**Comment by AlexSCorey** [comment]
Lets add some unit tests and some e2e tests.

**Review by AlexSCorey — CHANGES_REQUESTED** [review]
See comments:

**Comment by skochay330** [comment]
> The status on the list item should be a label rather than just an icon and text. https://tower-mockups.testing.ansible.com/patternfly/notifications/notifications/

@AlexSCorey I am using the standard `StatusCell` component in ITableColumn- this keeps it consistent with all other status cells across the platform

**Review by daphnemaeve — APPROVED** [review]

File: frontend/awx/administration/notifiers/hooks/useNotifiersFilters.tsx [hunk 1]
```diff
@@ -0,0 +1,10 @@
+import { useDynamicToolbarFilters } from '../../../common/useDynamicFilters';
+
+export function useNotifiersFilters() {
+  const toolbarFilters = useDynamicToolbarFilters({
+    optionsPath: 'notification_templates',
+    preSortedKeys: ['name'],
+    preFilledValueKeys: ['name', 'id'],
```
Comment (kialam) [inline frontend/awx/administration/notifiers/hooks/useNotifiersFilters.tsx:7]: The old UI has additional filters: Created by and Modified by. We should add these as well.

Comment (skochay330) [reply frontend/awx/administration/notifiers/hooks/useNotifiersFilters.tsx:7]: Good catch!!

**Review by AlexSCorey — APPROVED** [review]
This looks good to me.  Well done!

**Pull request merged**